source/manual/zendesk.html.md

```diff
@@ -60,26 +21,56 @@ GOV.UK have [committed to a minimum service level agreement](https://www.gov.uk/
 - 80% of general enquiries (public tickets) resolved within 5 working days
 - 70% of department tickets are closed within 5 working days
 
-## Technical 2nd Line
+## Get started
 
-### Get started
+[Create an account](https://govuk.zendesk.com/auth/v2/login/registration?auth_origin=3194076%2Cfalse%2Ctrue&amp;brand_id=3194076&amp;return_to=https%3A%2F%2Fgovuk.zendesk.com%2Fhc%2Fen-us&amp;theme=hc) then assign a new ticket to `2nd/3rd Line--Zendesk Administration` asking to give you "GDS Resolver Agent" access to [your product team's Zendesk group](#where-to-triage-tickets).
+
+See the [Zendesk best practice slidedeck](https://docs.google.com/presentation/d/1iUbD-_uWyaNMeNj9h7Zvo9g2GWvRarg9kUh7pd0u32M/edit#slide=id.g134fafb13dc_0_0) for an overview and tour of the service.
+
+In addition, you may want to bookmark these links for searching Zendesk:
+
+- [Tickets you have commented on](https://govuk.zendesk.com/agent/search/1?type=ticket&q=commenter%3Ame)
+- Read the [Zendesk documentation](https://support.zendesk.com/hc/en-us/articles/4408835086106-Using-Zendesk-Support-advanced-search) for advanced search options
 
-[Create an account](https://govuk.zendesk.com/auth/v2/login/registration?auth_origin=3194076%2Cfalse%2Ctrue&amp;brand_id=3194076&amp;return_to=https%3A%2F%2Fgovuk.zendesk.com%2Fhc%2Fen-us&amp;theme=hc) then assign a new ticket to `2nd/3rd Line--Zendesk Administration` asking to give you "GDS Resolver Agent" access to `2nd Line--GOV.UK Alerts and Issues`.
+## Where to triage tickets
 
-When you've logged in, navigate to the `2nd Line--GOV.UK Alerts and Issues` view (see [Summary of GOV.UK Zendesk groups](#summary-of-govuk-zendesk-groups)).
+Triage to the most applicable area of GOV.UK using the associated macro below. This is also represented visually on the [Zendesk flow diagram](https://docs.google.com/presentation/d/1EotoM2CVtqlnx54Qz5bP7OyIx5c9ji_GptUuymHkBrc/edit).
 
-### Workflow
+| Zendesk view | Macro for triaging to this view | Responsible for |
+|--------------|---------------------------------|----------|
+| [GOV.UK Publishing Service Support](https://govuk.zendesk.com/agent/filters/5273818481554) | [Triage to GOV.UK Publishing Service Support](https://govuk.zendesk.com/admin/workspaces/agent-workspace/macros/13679777581980) | Publishing apps, unpublishing content, etc |
+| [GOV.UK Web Support](https://govuk.zendesk.com/agent/filters/360000012465) | [Triage to GOV.UK Web Support](https://govuk.zendesk.com/admin/workspaces/agent-workspace/macros/13679771783708) | User-facing GOV.UK, GOV.UK search, GOV.UK chat, postcode lookup, etc  |
```

Comment:
You might want to add email alerts here, as it's a pretty common one coming to web